course3/homework-string/my_string.cc

```diff
@@ -0,0 +1,105 @@
+#include <iostream>
+#include <cstring>
+#include "my_string.h"
+
+MyString::MyString(){
+	buffer = std::unique_ptr<char[]>(new char[1]);
```

Comment:
This is correct, but it would be "nicer" if the default constructor wouldn't perform any allocations.